### PR TITLE
Added documentation for a query config

### LINKED ISSUE #8029: Document the query.info-url-template configuration property
Allow redirecting clients to an alternative location to fetch query information. This can be configured via the query.info-url-template configuration property. (#7678)

### PR DESCRIPTION
**Pull request by tan31989**
## Description

This PR adds documentation for the config that was introduced.

## Non-technical explanation
Updates the internal documentation to provide details on query info URI template configuration.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Closes the ({issue}`8029`)
```

Closes #8029

### CODE REVIEW

File: docs/src/main/sphinx/admin/properties-query-management.rst [hunk 1]
```diff
@@ -183,6 +183,20 @@ Timeout value for remote tasks that fail to communicate with the
 coordinator. If the coordinator is unable to receive updates from a remote task
 before this value is reached, the coordinator treats the task as failed.
 
+``query.info-url-template``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** (URI path that renders query details using internal query info page)
+
+This configuration allows redirecting clients to an alternative location in order to
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/properties-query-management.rst:192]: How about:

```
Configure redirection of clients to an alternative location for query information. The URL must contain a query id placeholder ``${QUERY_ID}``, for example ``https://example.com/query/${QUERY_ID}``. 
```

Ideally we should also add why/when you would use this .. could you explain?

File: docs/src/main/sphinx/admin/properties-query-management.rst [hunk 2]
```diff
@@ -183,6 +183,20 @@ Timeout value for remote tasks that fail to communicate with the
 coordinator. If the coordinator is unable to receive updates from a remote task
 before this value is reached, the coordinator treats the task as failed.
 
+``query.info-url-template``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/properties-query-management.rst:187]: Shorten to same length as 186

File: docs/src/main/sphinx/admin/properties-query-management.rst [hunk 3]
```diff
@@ -183,6 +183,20 @@ Timeout value for remote tasks that fail to communicate with the
...
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** (URI path that renders query details using internal query info page)
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/properties-query-management.rst:190]: Maybe

URL of the query info page on the coordinator

**Review by mosabua — CHANGES_REQUESTED** [review]

**Comment by mosabua** [comment]
This file was converted to markdown. Please rebase and apply the changes to the new file.

File: docs/src/main/sphinx/admin/properties-query-management.rst (hunk 1 above)
Comment (tan31989) [reply docs/src/main/sphinx/admin/properties-query-management.rst]: I have added the change, but I believe, the "why/when you would use this", while is good to have, the [PR](https://github.com/trinodb/trino/pull/7678) contributed towards its feature implementation, doesn't mention the need for the required config.

**Review by mosabua — APPROVED** [review]

**Pull request merged**